go.mod

```diff
@@ -26,8 +26,11 @@ require (
 	k8s.io/apiextensions-apiserver v0.25.0
 	k8s.io/apimachinery v0.25.0
 	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
+	sigs.k8s.io/yaml v1.3.0
 )
 
+replace github.com/codeready-toolchain/api => github.com/rajivnathan/api v0.0.0-20231031184215-7cfe76df1138
```

Comment:
I would recommend separating the go.mod dependency update from the CRD update:
- Merge api & the CRD PRs as soon as they both green.
- Create another PR for the go.mod update.
Otherwise there is a risk that the CRD + go.mod PR is stuck (for example there is an infra issue) while the api PR has been already merged. Which could potentially block others from updating api.

But it's minor. I will let you to decide how you want to proceed :)

Reply:
Yeah, good point. I'll do the go.mod update separately.